hidtest/test.c

```diff
@@ -131,6 +131,9 @@ int device_callback(
     printf("  Usage (page): 0x%hx (0x%hx)\n", device->usage, device->usage_page);
     printf("\n");
 
+	/* Printed data might not show on the screen - force it out */
+	fflush(stdout);
```

Comment:
the spaces are inconsistent in this function
I get that it is not directly related to your change, but do you mind replacing spaces with tabs in the entier function to make it consistent?
thanks

Reply:
Will do